modeshape-jcr/src/main/java/org/modeshape/jcr/RepositoryConfiguration.java

```diff
@@ -1211,8 +1212,17 @@ public BinaryStore getBinaryStore() throws Exception {
                 String username = binaryStorage.getString(FieldName.USER_NAME);
                 String password = binaryStorage.getString(FieldName.USER_PASSWORD);
                 String bucketName = binaryStorage.getString(FieldName.BUCKET_NAME);
-                store = new S3BinaryStore(username, password, bucketName);
+
+                //Use S3 provided endpoints
+                if (FieldName.ENDPOINT_URL != null) {
```

Comment:
I see what you meant with the constant... fixing shortly.  Thx!  I see what you meant with the constant... fixing shortly.  Thx!